Sources/Error Handling/ErrorUtils.swift

```diff
@@ -513,9 +514,12 @@ private extension ErrorUtils {
 
     // swiftlint:disable:next function_body_length
     private static func logErrorIfNeeded(_ code: ErrorCode,
+                                         message: String?,
                                          fileName: String = #fileID,
                                          functionName: String = #function,
                                          line: UInt = #line) {
+        let message = code.description + (message.map { " " + $0 } ?? "")
```

Comment:
Hmm maybe rename this to `formattedMessage` or something like that? so we don't shadow the `message` parameter.